src/Turbo/Bridge/Mercure/TurboStreamListenRenderer.php

```diff
@@ -44,10 +58,10 @@ public function renderTurboStreamListen(Environment $env, $topic): string
                 throw new \LogicException(sprintf('Cannot listen to entity of class "%s" as the PropertyAccess component is not installed. Try running "composer require symfony/property-access".', $class));
             }
 
-            $topic = sprintf(Broadcaster::TOPIC_PATTERN, rawurlencode($class), rawurlencode(implode('-', $id)));
+            $topic = sprintf(Broadcaster::TOPIC_PATTERN, rawurlencode($class.(\is_string($scope) ? '_'.$scope : '')), rawurlencode(implode('-', $id)));
```

Comment:
```suggestion
            $topic = sprintf(Broadcaster::TOPIC_PATTERN, rawurlencode($class.(null === $scope ? '' : '_'.$scope)), rawurlencode(implode('-', $id)));
```

Reply:
It might be worth adding a public static function to `Broadcaster` to help generate this string wherever we need to.

Reply:
Resolving this since it's no longer relevant with the new approach.